src/plugins/deploy/azureDeployPlugin.ts

```diff
@@ -16,19 +16,18 @@ export class AzureDeployPlugin {
     const functionAppService = new FunctionAppService(this.serverless, this.options);
     const functionApp = await functionAppService.get();
 
-    if (functionApp) {
-      await functionAppService.cleanUp(functionApp);
-    }
+    this.serverless.cli.log("NOTHING to do before deploy")
+    // TODO: maybe this is the place to check if all relevant files exist before uploading
   }
 
   private async deploy() {
     const resourceService = new ResourceService(this.serverless, this.options);
     await resourceService.deployResourceGroup();
 
     const functionAppService = new FunctionAppService(this.serverless, this.options);
-    const functionApp = await functionAppService.deploy();
 
-    await functionAppService.uploadFunctions(functionApp);
-    await functionAppService.syncTriggers(functionApp);
+    // create all necessary resources: resource-group, storage account, app service plan, and app service
```

Comment:
Is this comment saying that line 30 does all that? Or that this is where we _would_ do that?

Reply:
this is saying line 30 does all that